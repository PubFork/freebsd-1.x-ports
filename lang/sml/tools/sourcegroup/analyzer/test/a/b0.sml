group ["b1.sml", "bx.sml", "b2.sml"];
group ["b1.sml", "bx.sml", "b3.sml"];
