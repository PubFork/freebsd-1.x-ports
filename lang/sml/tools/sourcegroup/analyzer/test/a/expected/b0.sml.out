source sml "a/b0.sml";

